modules/AddBook.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/prefer-default-export */
+import { myLibrary } from './Library.js';
+
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+export const addBook = (e) => {
+  e.preventDefault();
+  if (bookTitle.value === '' || bookAuthor.value === '') {
+    return;
+  }
+  myLibrary.addBook(bookTitle, bookAuthor);
+  localStorage.setItem('book-collection', JSON.stringify(myLibrary.library));
+  bookTitle.value = '';
+  bookAuthor.value = '';
+  window.location.reload();
+};
```

Comment:
- [ ] (Optional) Remember that this application is meant to be a Single Page Application (SPA). Using `window.relocation.reload()` after adding a book will defeat the purpose. I would advise you to either toggle the classlist or set the add book section to display none and the list book section to display block. 👍